app.py

```diff
@@ -292,9 +297,14 @@ def parse():
     else:
         try:
             hedy_errors = TRANSLATIONS.get_translations(lang, 'HedyErrorMessages')
+            gradual_feedback = TRANSLATIONS.get_translations(lang, 'GradualFeedback')
             with querylog.log_time('transpile'):
                 result = hedy.transpile(code, level)
             response["Code"] = "# coding=utf8\nimport random\n" + result
+            if lang in supported_lang:
+                response['prev_feedback_level'] = session['error_level']
+                response['prev_similar_code'] = session['similar_code']
+                session['error_level'] = 0  # Code is correct: reset error_level back to 0
```

Comment:
Don't know what happened in the merges, but we need to use the `set_session_var` function. Here's the reference: https://github.com/Felienne/hedy/blob/test/app.py#L332

Reply:
Fixed!